grails-app/domain/org/pih/warehouse/inventory/CycleCountRequest.groovy

```diff
@@ -21,6 +24,28 @@ class CycleCountRequest {
 
     Boolean blindCount
 
+    /**
+     * The person who is responsible for performing the count. Note that this might be different from
+     * the person who *actually* performs the count.
+     */
+    Person countAssignedTo
+
+    /**
+     * The date that the count should be performed by.
+     */
+    LocalDate countDeadline
+
+    /**
+     * The person who is responsible for performing the recount. Note that this might be different from
+     * the person who *actually* performs the recount.
+     */
+    Person recountAssignedTo
+
+    /**
+     * The date that the recount should be performed by.
+     */
+    LocalDate recountDeadline
```

Comment:
I know these are different from what was in the ticket so we can discuss column names if you want